cli/validate-wca.js

```diff
@@ -0,0 +1,66 @@
+import chalk from 'chalk';
+import { fileURLToPath } from 'node:url';
+import path from 'node:path';
+import { readFile } from 'node:fs/promises';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const customElementsPath = path.join(__dirname, '../custom-elements.json');
+
+function validateTag(tag) {
+
+	// assume an element with no description is to be ignored
+	if (!tag.description) {
+		return 0;
+	}
+
+	let errors = 0;
+
+	tag.attributes?.forEach(a => {
+		if (!a.description) {
```

Comment:
I'm guessing there's no way that the description could be an empty string, ya?

Reply:
This will error when description is `undefined` or an empty string.

Reply:
Oh right. I don't know why I always thing empty string needs it's own check. 🤷‍♂️